Apps/WebClient/src/ClientApp/src/views/landing.vue

```diff
@@ -359,17 +386,17 @@ export default class LandingView extends Vue {
         },
         {
             definition: "syringe",
-            label: "Immunizations (Coming soon)",
+            label: "Immunizations",
             active: false,
         },
         {
             definition: "flask",
-            label: "Lab Results (Coming soon)",
-            active: false,
+            label: "Lab Results",
+            active: true,
```

Comment:
should this Lab flag be "true" by default as all the other active flags?

Reply:
defaulted all to false